src/runtime/TraceurLoader.js

```diff
@@ -59,17 +59,9 @@ export class TraceurLoader extends Loader {
   }
 
   locate(load) {
-    var normalizedModuleName = load.normalizedName;
+    var asJS = load.normalizedName;
```

Comment:
Since we no longer convert to .js, `asJS` does not make sense. How about `loadName` or `normalizedModuleName`?